bad_code.rb

```diff
@@ -0,0 +1,5 @@
+def bad_name
+  if something
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.